Katello/katello
# Fixes #32422 - cannot create repo with http proxy with auth creds

Moves the HTTP proxy username and password out of the URL and into the RpmRpmRemote parameters.

To test:

1) Ensure you're on Pulpcore 3.11
2) Create an HTTP proxy that has a username and password
3) Create a repository with that HTTP proxy attached and see the error in the redmine
